pkg/agent/datamodel/const.go

```diff
@@ -119,17 +119,15 @@ const (
 
 const (
 	Nvidia470CudaDriverVersion = "cuda-470.82.01"
-	Nvidia510CudaDriverVersion = "cuda-510.47.03"
-	Nvidia525CudaDriverVersion = "cuda-525.85.12"
-	Nvidia510GridDriverVersion = "grid-510.73.08"
+	Nvidia535CudaDriverVersion = "cuda-535.54.03"
 	Nvidia535GridDriverVersion = "grid-535.54.03"
 )
 
 // These SHAs will change once we update aks-gpu images in aks-gpu repository. We do that fairly rarely at this time
 // So for now these will be kept here like this.
 const (
 	AKSGPUGridSHA = "sha-20ffa2"
-	AKSGPUCudaSHA = "sha-16fd35"
```

Comment:
should we also be updating this in the build scripts?

Reply:
updated, will run a test build